pkgs/applications/window-managers/weston/default.nix

```diff
@@ -1,21 +1,25 @@
-{ lib, stdenv, fetchurl
+{ lib, stdenv, fetchFromGitLab
 , meson, ninja, pkg-config, python3, wayland-scanner
 , cairo, colord, dbus, lcms2, libGL, libXcursor, libdrm, libevdev, libinput
 , libjpeg, seatd, libxcb, libxkbcommon, mesa, mtdev, pam, udev, wayland
 , wayland-protocols
 , pipewire ? null, pango ? null, libunwind ? null, freerdp ? null, vaapi ? null
 , libva ? null, libwebp ? null, xwayland ? null
 # beware of null defaults, as the parameters *are* supplied by callPackage by default
-, buildDemo ? false
+, buildDemo ? true
+, buildRemoting ? true, gst_all_1
```

Comment:
Upstream  enabled build demo and remoting by defaul

Reply:
How about turning these off for `westonLite`, and having them on for `weston`?